src/app/submission/sections/clarin-notice/clarin-notice.component.spec.ts

```diff
@@ -102,7 +102,7 @@ describe('SubmissionSectionClarinNoticeComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(SubmissionSectionClarinNoticeComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
+    // fixture.detectChanges();
```

Comment:
?

Reply:
It is a hack to pass this unit test. It is used in more places.